timur/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.4"
+volumes:
+  app-gems:
+    driver: local
+  timur-db-data:
```

Comment:
Namespace the volumes by app name, so we don't run into conflicts with the other containers.

Reply:
:+1: